EDDI/App.xaml.cs

```diff
@@ -19,12 +19,12 @@ static void Main()
             // Configure Rollbar error reporting
 
             // Generate or retrieve an id unique to this configuration for bug tracking
-            string uniqueId = Eddi.Properties.Settings.Default.uniqueID ?? Guid.NewGuid().ToString();
-            if (Eddi.Properties.Settings.Default.uniqueID == null)
+            if (Eddi.Properties.Settings.Default.uniqueID == null ||
```

Comment:
There's a convenience method `string.IsNullOrEmpty(Eddi.Properties.Settings.Default.uniqueID)` but this is fine too.